token/core/fabtoken/transfer.go

```diff
@@ -125,14 +136,15 @@ func (s *TransferService) Transfer(txID string, wallet driver.OwnerWallet, ids [
 	}
 
 	metadata := &driver.TransferMetadata{
-		Outputs:            outputs,
```

Comment:
I think it would give more clarity to have a struct:
```
type OutToken struct {
 TokenID
 Sender
 Output
 Metadata
 AuditInfo
 Receiver
 ReceiverAuditInfo
 ReceiverIsSender
}
```

This way it's not necessary to read the code to figure out which slice elements are related, and we don't need to check that the lengths are correct.

Reply:
👍